Pinta.Effects/Effects/JuliaFractalEffect.cs

```diff
@@ -93,7 +93,7 @@ private JuliaSettings CreateSettings (ImageSurface dst)
 			aspect: (double) h / (double) w,
 			count: count,
 			invCount: 1.0 / (double) count,
-			angleTheta: (Data.Angle * Math.PI * 2) / 360.0,
+			angleTheta: (Data.Angle.Degrees * Math.PI * 2) / 360.0,
```

Comment:
There are some effects which always convert the value to radians. Is your plan to only support DegreesAngle in SimpleEffectDialog, and then update effects to use the conversion functions to RadiansAngle instead of manual expressions like this?

Reply:
Yes, my plan is to eventually use the appropriate angle types for all operations, but for now the priority was on the public-facing part of this